src/jit/inline.cpp

```diff
@@ -474,6 +474,12 @@ void InlineContext::DumpXml(FILE* file, unsigned indent)
         m_Sibling->DumpXml(file, indent);
     }
 
+    // Optionally suppress failing inline records
+    if ((JitConfig.JitInlineDumpXml() == 3) && !m_Success)
+    {
```

Comment:
Should we add an enum for the three values used by JitInlineDumpXml()

Reply:
There are a bunch of enhancements I have in mind for the inline XML, though as far as I know nobody but me has ever used it.... also I have some analysis tooling lying about that I should publish somewhere, maybe over in jitutils.

I should really change this into a flags-style thing.... you should be able to choose:
* get XML for all methods or only those with inlines or inline attempts
* get XML for all inline attempts or only those with successful inlines
* get XML with full details or just summary details
* specify filename to contain the results (currently always goes to stderr)

So maybe this is best done as a follow-up change.